wg-infra/policy/joining-governance.md

```diff
@@ -0,0 +1,13 @@
+# Joining Electron Governance
+
+This documents the process for becoming a member of Electron Governance. Being a member of Electron Governance means you are actively a member of at least one Working Group.
+
+## Requirements
+
+* A new member is approved by their prospective Working Group according to their rules for membership. Approval is recorded in meeting notes.
+* A new member has a GitHub account
+* A new member is a guest on Electron's Slack
+
+## Process
+
+The process for onboarding a new member into Electron Governance is documented and governed via a workflow in the #wg-infra channel.  To use it please ensure the above requirements are met and then trigger the `Onboard Maintainer` workflow.  This workflow will guide you through the process and ping the appropriate teams when required.
```

Comment:
```suggestion
The process for onboarding a new member into Electron Governance is documented and governed via a workflow in the #wg-infra channel.  To use it please ensure the above requirements are met and then trigger the `Onboard Maintainer` workflow. This workflow will guide the onboarding process and ping the appropriate teams when required.
```

i’d also heavily prefer that even if we’re delegating to the onboarding workflow, we still enumerate the steps the bot takes for reference and transparency.